eng/Directory.Build.Data.props

```diff
@@ -85,8 +85,8 @@
 
   <PropertyGroup Condition="'$(IsTestProject)' == 'true' or '$(IsTestSupportProject)' == 'true' or '$(IsSamplesProject)' == 'true' or '$(IsPerfProject)' == 'true' or '$(IsStressProject)' == 'true'">
     <IsPackable>false</IsPackable>
-    <RequiredTargetFrameworks>netcoreapp2.1</RequiredTargetFrameworks>
-    <RequiredTargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netcoreapp2.1;net461</RequiredTargetFrameworks>
+    <RequiredTargetFrameworks>netcoreapp2.1;net5</RequiredTargetFrameworks>
```

Comment:
Given some discussions on Twitter about this (with Immo et. al.) we might want to use "net5.0" here. "net5" works, but in another 5 years or so may pose problems.